api/api/src/test/scala/uk/ac/wellcome/platform/api/services/FiltersAndAggregationsBuilderTest.scala

```diff
@@ -91,6 +91,28 @@ class FiltersAndAggregationsBuilderTest extends AnyFunSpec with Matchers {
         .subaggs should have length 0
     }
 
+    it("applies paired filters to non-paired aggregations") {
+      val formatFilter = FormatFilter(Seq("bananas"))
+      val sut = new FiltersAndAggregationsBuilder(
```

Comment:
`sut`?

Reply:
It's what I used in the other tests in Jan so just made it match, it's a rather old fashioned term for [System Under Test](https://en.wikipedia.org/wiki/System_under_test)

Reply:
I think I got it from here maybe https://docs.docker.com/docker-hub/builds/automated-testing/